src/lib.rs

```diff
@@ -179,6 +193,13 @@ mod tests {
         assert_eq!(Some(TIMESTAMP), result);
     }
 
+    #[pg_test]
+    fn test_timestamp_to_ulid() {
+        let result =
+            Spi::get_one::<&str>(&format!("SELECT '{TIMESTAMP}'::timestamp::ulid::text;")).unwrap();
+        assert_eq!(Some(TIMESTAMP_ULID_PREFIX), result);
```

Comment:
You don't need to define the constant and instead of use the value here directly.

Reply:
Done